### PR TITLE
Misc: skip checking for incorrectly formatted jsdoc blocks

### PR DESCRIPTION
**Pull request by zxqfox**
Fixes jscs-dev/node-jscs#1588

### CODE REVIEW

**Comment by zxqfox** [comment]
@Nightmaster @mrjoelkemp any response, guys?

File: lib/rules/validate-jsdoc.js [hunk 1]
```diff
@@ -122,7 +122,7 @@ module.exports.prototype = {
                 }
 
                 validators = _this._rulesForTags[scope];
-                if (!validators || !node.jsdoc) {
+                if (!validators || !node.jsdoc || !node.jsdoc.valid) {
```
Comment (mrjoelkemp) [inline lib/rules/validate-jsdoc.js]: This has to be done for every jsdoc rule? :cry:

**Comment by mrjoelkemp** [comment]
:+1:

File: lib/rules/validate-jsdoc.js (hunk 1 above)
Comment (zxqfox) [inline lib/rules/validate-jsdoc.js]: Didn't find a better way to split rules into files ;-(

But feels like it will be changed soon.

**Comment by zxqfox** [comment]
Thanks, Joel!

**Pull request closed without merging**